src/libsync/account.h

```diff
@@ -304,10 +304,13 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject
     void setE2eEncryptionKeysGenerationAllowed(bool allowed);
     [[nodiscard]] bool e2eEncryptionKeysGenerationAllowed() const;
 
+    [[nodiscard]] bool askUserForMnemonic() const;
```

Comment:
why not also making it a real Qt property ?

Reply:
I didn't think it necessary -- made it one now, however